tools/build_cros_gcc

```diff
@@ -104,20 +107,29 @@
    $0 $1 $2 $(crew const CREW_PREFIX | cut -d'=' -f2)
    exit 0
  fi
+ if test ! $4; then
```

Comment:
This should probably be `if test $3; then`.  `if test ! $4; then` will always be null even if $3 is not provided.

Reply:
Yeah. That's what I'm working on. Stay tuned.